pyro/distributions/testing/gof.py

```diff
@@ -0,0 +1,280 @@
+# Copyright (c) 2014, Salesforce.com, Inc.  All rights reserved.
```

Comment:
This file is ported from https://github.com/posterior/goftests/blob/master/goftests/__init__.py
Differences include:
- this restricts to continuous distributions
- this ports to PyTorch and Python 3